discojs/src/aggregator/get.ts

```diff
@@ -33,6 +34,42 @@ export function getAggregator(
   const scheme = options.scheme ?? task.trainingInformation.scheme
   
   switch (task.trainingInformation.aggregationStrategy) {
+    case 'byzantine': {
+        const {
+          clippingRadius = 1.0,
+          maxIterations = 1,
+          beta = 0.9,
+        }:{
+            clippingRadius?: number;
+            maxIterations?: number;
+            beta?: number;
+          } = task.trainingInformation;
+
+        if (scheme === "decentralized") {
+          options = {
+            roundCutOff: undefined,
+            threshold: 1,
+            thresholdType: "relative",
+            ...options,
+          };
+        } else {
+          options = {
+            roundCutOff: undefined,
+            threshold: 1,
+            thresholdType: "absolute",
+            ...options,
+          };
+        }
```

Comment:
duplicated from the "mean" case, consider extracting it into a dedicated variable, smth like
```ts
const networkOptions: Required<AggregatorOptions> = {
	scheme,
	roundCutOff: 0,
	threshold: 1,
	thresholdType: scheme === "decentralized" ? "relative" : "absolute",
	...options
};
```